api/server/httputils/errors.go

```diff
@@ -54,6 +54,7 @@ func GetHTTPErrorStatusCode(err error) int {
 			code    int
 		}{
 			{"not found", http.StatusNotFound},
+			{"cannot find", http.StatusNotFound},
```

Comment:
Ok, is there an issue about this? I don't think we return a 404 on other platforms when  the path to a binary is incorrect.